src/Collection/Helpers/ArrayCollectionHelper.php

```diff
@@ -11,6 +11,8 @@
 use Closure;
 use DateTimeImmutable;
 use DateTimeInterface;
+use http\Exception\InvalidArgumentException;
```

Comment:
Typo? Phpstorm love this class.

Reply:
I do PHP not much nowadays, what the hell was this? 😨 Some PhpStorm's bug?

Reply:
Idk. In last version it displays this exception always first instead of namespace specific one.